src/Models/Services/OatPublicUtils.ts

```diff
@@ -14,7 +15,7 @@ import {
 import { convertModelToDtdl, parseModelId, safeJsonParse } from './OatUtils';
 import { deepCopy, getDebugLogger, isDefined } from './Utils';
 
-const debugLogging = false;
+const debugLogging = true;
```

Comment:
I think we want to keep this false

Reply:
damn! we really ned to build the eslint rule for this.